Remotion/Web/Core/UI/Controls/CommandInfo.cs

```diff
@@ -157,7 +157,22 @@ private void AddDiagnosticMetadataAttributes (HtmlTextWriter writer)
 
     private bool IsTriggeringNavigation ()
     {
-      return _href != "#";
+      if (string.IsNullOrEmpty(_onClick))
+      {
+        return true;
+      }
+
+      if (_onClick.Contains("__doPostBack"))
+      {
+        return false;
+      }
+
+      if (_href == null || _href.StartsWith("#"))
+      {
+        return false;
+      }
+
+      return true;
     }
 
     private bool IsTriggeringPostBack ()
```

Comment:
This also needs an update now. 
```
  return _onClick?.Contains("__doPostBack");
```

Why not do it as separate if/elseif/else ? We can, it just seems overkill because there's just the simple contains check. But yes, doing this as separate statements would be the more consistent style so go for it :)